airbyte-integrations/connectors/source-close-com/source_close_com/source.py

```diff
@@ -91,7 +91,9 @@ class CloseComStreamCustomFields(CloseComStream):
 
```

Comment:
Looking at this file, I would love to migrate the whole thing to low-code, I think this should already be available in the Builder. Any takers? Is this on our list?

Reply:
This is in the migrate to builder list, but it is not compatible because, for incremental sync, it requires different formats for records, sometimes its %Y-%m-%sTHH:MM:SS, sometimes  %Y-%m-%sTHH:MM:SSZ tor other formats

Reply:
I _think_ we should support this already, but not sure. So what you're saying is that the date could be in different formates? /cc @maxi297, do you know if incremental low code would eat that?

Reply:
Oh! I was not aware of that improvement, I guess it will make this connector compatible with builder, I'll check and update soon

Reply:
Thanks @maxi297 for this insight✨